lib/builders-with-pkgs.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, pkgs }:
+with lib.nixvim.builders;
+rec {
```

Comment:
what makes this recursive? i don't see it at first glance

Reply:
I don't think it is anymore (the old file was). I'll see if I can remove it.

Reply:
Refactored the implementation to be more dynamic. This file is now removed.